src/locutus/api/__init__.py

```diff
@@ -1,4 +1,4 @@
-from sessions import SessionManager
+from locutus.sessions import SessionManager
 from locutus import get_code_index
```

Comment:
<img width="322" alt="image" src="https://github.com/user-attachments/assets/b7dcf98c-4b62-406c-9f7b-25a93347c4d9" />
 
I'm not exactly sure why GH thinks 'main' has `from sessions`. The screenshot above shows what is currently in `main`,  I probably had a merge conflict and didn't see that it was changing the sessions import as well as adding the code index import. I suspect this edit will disappear between now and after the merge, since there were no changes.

This note applies to changes in[ this commit ](https://github.com/NIH-NCPI/locutus/pull/110/commits/a674c0f9bf28c34a5a415d9b34d4ffba40673062)